ACT Landing Page.md

```diff
@@ -0,0 +1,173 @@
+
+### The ACT Difference
+
+While traditional assessments, such as Security Control Assessments (SCAs), generate valuable lists of low-level technical findings, ACT **transforms** this data into **actionable intelligence**:
+
+- **Risk-driven rather than compliance-driven**: ACT focuses on risk identification and analysis at the capability level, de-emphasizing technical findings and compliance checkboxes
+- **Capability-oriented rather than control-oriented**: Capabilities state objectives, while controls state specific implementation requirements that might help meet those objectives
+- **Based on multiple Risk Information Sources**: ACT considers **all available risk data** at the time of assessment, not just current compliance status
+- **More understandable and actionable**: ACT adds context, brings conversation to a higher level, and helps readers determine **what should be done**
+
+---
+
+## Why ACT Matters
+
+### The Problem with Traditional Assessments
+
+Many federal and commercial cybersecurity programs suffer from similar weaknesses:
+
+1. Low-level technical findings are handed directly to less-technical decision makers
+2. Decision-makers lack an appropriate framework for understanding the actual risks
+3. Without risk context, "close all findings" becomes the default reaction - **regardless of cost or impact to the organization**
+4. Organizations blindly comply with standards rather than make risk-based decisions
+5. **Money and resources are wasted on inappropriate compliance efforts**
+
+### The ACT Solution
+
+ACT addresses these challenges by:
+
+- Implementing a risk-based decision-making framework that is tailored to the needs of your specific organization, while maintaining compatibility and comparability with other organization's ACT implementations
+- Providing context that explains what technical findings mean to your mission
+- Empowering decision-makers to understand and choose when and how to comply, and when to accept risk
+- Focusing resources on the highest-impact security improvements
+- Supporting efficient, cost-effective, and repeatable risk-based assessment processes
+
+---
+
+## How ACT Works
+
+### Security Capabilities Focus
+
+ACT organizes security objectives into high-level Capabilities such as:
+
+- **CRED**: Manage Credentials and Authentication
+- **PRIV**: Manage Privileges and Accounts
+- **VULN**: Vulnerability (Patch) Management
+- **BEHAVE**: Manage Behavioral Expectations
+
+Each Capability (and their multiple Sub-Capabilities) reflects the high-level _why_ a system needs to be secure, rather than the low-level _how_ it complies.
```

Comment:
Oh cool you do use the `why`/`how` verbiage here. Do you think it's worth sharing an example here?